index.html

```diff
@@ -3,9 +3,18 @@
   <head>
     <meta charset="utf-8">
     <title>super-fantastic-dashboard</title>
+    <style>
+      body {
+        background-repeat: no-repeat;
+        background-position: center center;
+        background-attachment: fixed;
+        background-size: cover;
+      }
+    </style>
   </head>
   <body>
     <div id="app"></div>
     <script src="/dist/build.js"></script>
+    <script src="/static/js/background_image_loader.js"></script>
```

Comment:
@moltenkaizen I agree. Feels a bit uneasy having this outside of the Vue scope. Perhaps we can even try to make a component out of this or add it to the `App.vue` file. We could probably add these styles to the `#app` ID instead of `<body>`.

Then we could just add the functionality to the vue file instead of a function.